### PR TITLE
Ported code snippets to Python3

### PR DESCRIPTION
**Pull request by yashLadha**

### CODE REVIEW

**Comment by yashLadha** [comment]
Please review: @francisf

File: parallel_tests/browsers.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [
   {
-    "browserName": "iPhone",
```
Comment (Varun-garg) [inline parallel_tests/browsers.json]: Iphone 5s is no longer supported without emulators. Could you add a device from the list here? https://local.bsstag.com/list-of-browsers-and-platforms/automate . I would suggest use an  iPhone 8 with iOS 11 . These capabilities (`platform`) are also old, can you please add the ones generated here? https://www.browserstack.com/automate/capabilities

**Comment by Varun-garg** [comment]
This branch is for selenium 3, please also make the same changes in `selenium-4` branch.

File: parallel_tests/browsers.json (hunk 1 above)
Comment (francisf) [reply parallel_tests/browsers.json]: Plus changes  to selenium-4 branch

Comment (yashLadha) [reply parallel_tests/browsers.json]: Updated capabilities 👍

**Pull request merged**